src/elements/GridItem.vue

```diff
@@ -25,11 +25,19 @@ export default {
     },
     /**
      * Sets the size of the column. Prefix with `sm-` or `lg-`. Based on a 12 column grid.
+     * The inclusion of `auto` will set that grid item to have a width based on the width and height of the content.
      */
     columns: {
       type: String,
       default: "",
     },
+    /**
+     * Sets the vertical alignment of the item. `start`, `center`, or `end`. For horizontal alignment, please look at the wrapper component.
+     */
+    vertical: {
```

Comment:
I have to say that I'm a bit confused by the `horizontal` prop being on Wrapper and the `vertical` being on GridItem. If I wanted to align the GridItem content, I would have to use a Wrapper?

Reply:
There is no `justify-items` or `justify-self` property available on the main axis. The only property available for horizontal alignment in flexbox is `justify-content` which has to be applied to the parent/container of the flexbox item.

This setup is confusing and I'm wondering if we should extract the `flexContainer` out of the `Wrapper` element and create a `GridContainer` element.

Reply:
> This setup is confusing and I'm wondering if we should extract the flexContainer prop out of the Wrapper element and create a GridContainer element.

I'm a fan of this approach. I don't want to add extra work, but it seems like the simplest approach and semantically couples those two components so folks know they should be used together.